src/pages/GithubRedirectPage.tsx

```diff
@@ -20,9 +20,9 @@ const GithubRedirectPage = () => {
     workspace
       ? `/workspaces/${workspace.metadata.name}${
           application
-            ? `/applications/${application}${
-                task ? `/taskruns/${task}` : pipelineRun ? `/pipelineruns/${pipelineRun}` : ''
-              }${isLogsTabSelected ? `/logs` : ''}`
+            ? `/applications/${application}${pipelineRun ? `/pipelineruns/${pipelineRun}` : ''}${
+                isLogsTabSelected ? `/logs` : ''
+              }${task ? `?task=${task}` : ''}`
```

Comment:
when no logs tab, why do we add `?task=task` at the end of URL?

Reply:
task itself won't be available in the URL if `/logs/` is not present in the redirect URL